test/tests/neutronics/heat_source/particles_from_postprocessor.i

```diff
@@ -0,0 +1,90 @@
+[Mesh]
+  [sphere]
+    type = FileMeshGenerator
+    file = ../meshes/sphere.e
+  []
+  [solid]
+    type = CombinerGenerator
+    inputs = sphere
+    positions = '0 0 0
+                 0 0 4
+                 0 0 8'
+  []
+  [solid_ids]
+    type = SubdomainIDGenerator
+    input = solid
+    subdomain_id = '100'
+  []
+  [fluid]
+    type = FileMeshGenerator
+    file = stoplight.exo
+  []
+  [fluid_ids]
+    type = SubdomainIDGenerator
+    input = fluid
+    subdomain_id = '200'
+  []
+  [combine]
+    type = CombinerGenerator
+    inputs = 'solid_ids fluid_ids'
+  []
+[]
+
+[Postprocessors]
+  [p]
+    type = Receiver
+    default = 100.0
+  []
+  [particles]
+    type = ConstantPostprocessor
+    value = 100
+    execute_on = 'timestep_begin'
+  []
+[]
+
+[Problem]
+  type = OpenMCCellAverageProblem
+  power = p
+  particles = particles
+  temperature_blocks = '100 200'
+  density_blocks = '200'
+  verbose = true
+  cell_level = 0
+
+  initial_properties = xml
+
+  [Tallies]
```

Comment:
Please make the test as simple as possible - you can delete all tallies, you can turn off multiphysics feedback, etc. Will make it easier to maintain long-term.